### DISCUSSION TITLE
Any method for updating an entire table?

### DISCUSSION THREAD

**Discussion by arolluisma** [zauberzeug/nicegui#203] [Q&A] [answered]
The table object does not seem to have a method that would allow replacing its entire contents with another set of data. If data filtering/manipulation is done using an underlying dataframe, and we want the new result to be shown in the same Nicegui table, are there methods or properties, or in general is there a straightforward way of loading say a Pandas dataframe on to a table, to refresh the table's contents (i.e., in response to client-side inputs)?

Thanks for any suggestions.

**Comment by falkoschindler** [comment, marked as answer]
The idea is to modify `table.options` as you wish and call `table.update()` to communicate the changes to the client. As of version 1.0.7, this does not work in every case (see #204). But this should be fixed in the next release.

**Comment by rodja** [comment]
We just released [v1.0.8](https://github.com/zauberzeug/nicegui/releases/tag/v1.0.8) which fixes updating the table data.
